rdr_service/model/bq_participant_summary.py

```diff
@@ -34,7 +34,7 @@ class BQConsentCohort(Enum):
     COHORT_3 = 3  # New Participants with consent starting from April 17, 2020.
 
 COHORT_1_CUTOFF = date(2018, 4, 24)
-COHORT_2_CUTOFF = date(2020, 4, 16)
+COHORT_2_CUTOFF = date(2020, 4, 20)
```

Comment:
the comment on line 33 above doesn't reflect the new cutoff date.

Reply:
done